### PR TITLE
SWI-3498

### PR DESCRIPTION
**Pull request by brianluisgomez**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Global rule:
+* @Bandwidth-Samples/dx
```
Comment (ckoegel) [inline .github/CODEOWNERS]:
```suggestion
* @Bandwidth-Samples/band-swi 
```

File: .github/workflows/on-pull-request.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    branches:
+      - main
+
+jobs:
+  maven-test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```
Comment (ckoegel) [inline .github/workflows/on-pull-request.yaml]:
```suggestion
        uses: actions/checkout@4
```

File: README.md [hunk 3]
```diff
@@ -1 +1,78 @@
...
+To create an application log into the [Bandwidth Dashboard](https://dashboard.bandwidth.com/) and navigate to the `Applications` tab.  Fill out the **New Application** form selecting the service (Messaging or Voice) that the application will be used for.  All Bandwidth services require publicly accessible Callback URLs, for more information on how to set one up see [Callback URLs](#callback-urls).
+
+For more information about API credentials see our [Account Credentials](https://dev.bandwidth.com/docs/account/credentials) page.
+
+# Running the Application
+
+To install the required packages for this app, run the command:
+
+```sh
+pip install -r reaquirements.txt
```
Comment (ckoegel) [inline README.md]: this should be java specific

File: README.md [hunk 4]
```diff
@@ -1 +1,78 @@
...
+```
+
+Use the following command to run the application:
+
+```sh
+python main.py
```
Comment (ckoegel) [inline README.md]: same here

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**